portal-api/src/v1/gef/services/clone/helpers/generate-cloned-facility.test.ts

```diff
@@ -0,0 +1,131 @@
+import { CURRENCY, Facility, FACILITY_TYPE } from '@ukef/dtfs2-common';
+import { ObjectId } from 'mongodb';
+import { add } from 'date-fns';
+import { generateAuditDatabaseRecordFromAuditDetails, generateSystemAuditDetails } from '@ukef/dtfs2-common/change-stream';
+import { generateClonedFacility } from './generate-cloned-facility';
+
+const existingDealId = new ObjectId().toString();
+const newDealId = new ObjectId();
+const mockAuditDetails = generateSystemAuditDetails();
+
+const facilityDefaults = {
```

Comment:
is there a mock for this?

Reply:
Its not ideal, for similar reasoning to the deals